pipelines/jobs/configurations/jdk11u_pipeline_config.groovy

```diff
@@ -4,7 +4,7 @@ class Config11 {
                 os                  : 'mac',
                 arch                : 'x64',
                 additionalNodeLabels : 'macos10.14',
-                test                : ['sanity.openjdk', 'sanity.system', 'extended.system', 'sanity.perf'],
+                test                : 'default',
```

Comment:
I wonder if we need to define the default set in this file instead?  Else the reader has to dig into an unknow source file to figure that out...